build/targets/KoreBuild.Common.targets

```diff
@@ -53,6 +37,28 @@ as they are subject to change.
 
 <!--
 ###################################################################
+Targets: UseVolatileFeed
+
+Replaces $(RepositoryRoot)\NuGet.config feeds.
+###################################################################
+-->
+  <ItemGroup>
+    <UpdateFeeds Include="AspNetCore" Value="$(NUGET_VOLATILE_FEED_AspNetCore)" Condition="'$(NUGET_VOLATILE_FEED_AspNetCore)' != ''" />
```

Comment:
We add one for Universe `NUGET_VOLATILE_FEED_ARTIFACTS` (which is when it's trying to build against each other). We could just use an explicit list for now.